core/src/main/java/com/scalar/db/api/DistributedTransactionAdmin.java

```diff
@@ -0,0 +1,427 @@
+package com.scalar.db.api;
+
+import com.scalar.db.exception.storage.ExecutionException;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * An administrative interface for distributed transaction implementations. The user can execute
+ * administrative operations with it like createNamespace/createTable/getTableMetadata.
+ */
+public interface DistributedTransactionAdmin {
```

Comment:
This is a new API class, `DistributedTransactionAdmin`. With it, we can execute administrative operations for distributed transaction implementations.